l10n_it_fatturapa_out/wizard/wizard_export_fatturapa.py

```diff
@@ -32,22 +32,9 @@ class WizardExportFatturapa(models.TransientModel):
     _name = "wizard.export.fatturapa"
     _description = "Export E-invoice"
 
-    @api.model
-    def _domain_ir_values(self):
-        model_name = self.env.context.get("active_model", False)
-        # Get all print actions for current model
-        return [
-            ("binding_model_id", "=", model_name),
-        ]
-
-    def _get_selection(self):
-        reports = self.env["ir.actions.report"].sudo().search(self._domain_ir_values())
-        ret = [(str(r.id), r.name) for r in reports]
-        return ret
-
-    report_print_menu = fields.Selection(
-        selection="_get_selection",
-        help="This report will be automatically included in the created XML",
+    report_print_menu = fields.Many2one(
+        "ir.actions.report",
```

Comment:
Sembra un rollback di https://github.com/OCA/l10n-italy/commit/0315ba5e5a601765552bee38fce4d70a3fd78dcc, @TheMule71 @michelerusti cosa ne pensate?

Reply:
Ho visto anche la PR che ha portato da Many2one a Selection , ma secondo il mio parere ha più senso sia una many2one visto anche che nel caso di Selection, viene riportato errore se viene settato quel campo senza contesto active_model .... 
Cioè gli attributi dell'entità del wizard, non dovrebbero essere strict rispetto al contesto ma  il contesto dovrebbe solamente suggerire il report da utilizzare. Poi usando widget=selection anche la many2one diventa come selection e quindi trasparente per ,l'utente finale

Poi chi dovrà provvedere ad esportare le fatture, avrà sicuramente anche accesso al modello ir.actions.report quindi non dovrebbero esserci problemi di permessi d'accesso

Ovviamente parere mio tecnico, potrei anche sbagliarmi

Reply:
Quando si prova ad aggiornare il modulo in questione si riceve questo errore:

psycopg2.errors.DatatypeMismatch: foreign key constraint "wizard_export_fatturapa_report_print_menu_fkey" cannot be implemented
DETAIL:  Key columns "report_print_menu" and "id" are of incompatible types: character varying and integer.

Reply:
> Quando si prova ad aggiornare il modulo in questione si riceve questo errore:
> 
> psycopg2.errors.DatatypeMismatch: foreign key constraint "wizard_export_fatturapa_report_print_menu_fkey" cannot be implemented DETAIL: Key columns "report_print_menu" and "id" are of incompatible types: character varying and integer.

E' strano perchè è come se non riuscisse a cambiare il type del field da char a integer sul db . 
Odoo dovrebbe prima aggiungere i campi e poi passare alla fase di creazione delle fkey e indici.
Non sarà che il transient ha dei dati sopra e quindi non consente il cambio del tipo del campo  ?